charts/README.md

```diff
@@ -0,0 +1,63 @@
+# Deploy the Alert Logic Agent Container for Kubernetes using Helm 2
+
+This directory contains the al-agent Chart directory, which has the kubernetes manifests to deploy.
+
+## Before You Begin
+- You must have the kubectl and helm command line interfaces installed and get authentication credentials to interact with the cluster where you want to install the Agent Container.
+- Download the al-agent chart directory.
+- To deploy the Alert Logic Agent Container for Kubernetes, you need the unique registration key for your Alert Logic account.
```

Comment:
The key is not required for AWS and Azure deployments, we are actually going to audit all references to this within other README files and change the verbiage everywhere else. Until then, please remove this line or make it optional for non-AWS/Azure deployments.

Reply:
done